app/viewmodels/charityInformation/CharityInformationSummaryHelper.scala

```diff
@@ -80,35 +80,48 @@ class CharityInformationSummaryHelper(override val userAnswers: UserAnswers)
   ).flatten
 
   private def answerCharityContactDetails(charityContactDetails: CharityContactDetails,
-                                          changeLinkCall: Call)( implicit messages: Messages): Seq[SummaryListRow] = Seq(
-
-    Some(
-      summaryListRow(
-        label = messages("charityContactDetails.mainPhoneNumber.checkYourAnswersLabel"),
-        value = charityContactDetails.daytimePhone,
-        visuallyHiddenText = Some(messages("charityContactDetails.mainPhoneNumber.checkYourAnswersLabel")),
-        changeLinkCall -> messages("site.edit")
-      )
-    ),
-
-    Some(
+                                          changeLinkCall: Call)( implicit messages: Messages): Seq[SummaryListRow] = {
+    val seq= Seq(Some(
       summaryListRow(
         label = messages("charityContactDetails.alternativePhoneNumber.checkYourAnswersLabel"),
-        value = charityContactDetails.mobilePhone,
+        value = charityContactDetails.mobilePhone.getOrElse(""),
         visuallyHiddenText = Some(messages("charityContactDetails.alternativePhoneNumber.checkYourAnswersLabel")),
         changeLinkCall -> messages("site.edit")
       )
-    ),
+    ))
 
-    Some(
-      summaryListRow(
-        label = messages("charityContactDetails.emailAddress.checkYourAnswersLabel"),
-        value = charityContactDetails.emailAddress,
-        visuallyHiddenText = Some(messages("charityContactDetails.emailAddress.checkYourAnswersLabel")),
-        changeLinkCall -> messages("site.edit")
+    Seq(
+
+      Some(
+        summaryListRow(
+          label = messages("charityContactDetails.mainPhoneNumber.checkYourAnswersLabel"),
+          value = charityContactDetails.daytimePhone,
+          visuallyHiddenText = Some(messages("charityContactDetails.mainPhoneNumber.checkYourAnswersLabel")),
+          changeLinkCall -> messages("site.edit")
+        )
+      ),
+
+      if (charityContactDetails.mobilePhone.isDefined) {
```

Comment:
could you please change it with correct reference, this might help you instead of if else

      charityContactDetails.mobilePhone.map { mobilePhone =>
        summaryListRow(
          label = messages("charityContactDetails.alternativePhoneNumber.checkYourAnswersLabel"),
          value = mobilePhone,
          visuallyHiddenText = Some(messages("charityContactDetails.alternativePhoneNumber.checkYourAnswersLabel")),
          changeLinkCall -> messages("site.edit")
        )
      }